module.go

```diff
@@ -184,13 +184,12 @@ func loadSupressor(ctx *ntcontext, inp *device, out *device) error {
 	return nil
 }
 
-func loadModule(ctx *ntcontext, module, args string) (uint32, error) {
-	// TODO: CODE REMOVED
-	// MUST BE WRITTEN FROM SCRATCH WITHOUT LOOKING AT THE ORIGINAL CODE
-	// Description:
-	// Loads a module using pcClient.
-	// Then checks if it loaded correctly.
-	// Will display an error if any.
+func loadModule(ctx *ntcontext, module string, args string) (uint32, error) {
+	index, err := ctx.paClient.LoadModule(module, args)
+	if err != nil {
+		ctx.views.Push(makeErrorView(ctx, err.Error()))
+	}
+	return index, err
```

Comment:
I vote to leave this the way it is, since I don't actually know Go and I can't figure out how to get the error code out of the `err` variable. :)

And really we should display any error that occurs anyways, as there could maybe be others?

Reply:
Yes, we'll leave it this way. I don't know which other errors could happen here, NoiseTorch has many error checks for all sorts of things (which is a good thing), but I don't have a system with the flaws required to trigger those error checks and see if those error conditions work correctly.